pallets/multisig/src/lib.rs

```diff
@@ -249,31 +249,6 @@ decl_module! {
 
         fn deposit_event() = default;
 
-        fn on_runtime_upgrade() -> Weight {
-            use sp_version::RuntimeVersion;
-            use polymesh_primitives::migrate::kill_item;
-
-            // Kill pending proposals if the transaction version is upgraded
-            let current_version = <T::Version as Get<RuntimeVersion>>::get().transaction_version;
-            if TransactionVersion::get() < current_version {
-                TransactionVersion::set(current_version);
-                for item in &["Proposals", "ProposalIds", "ProposalDetail", "Votes"] {
-                    kill_item(NAME, item.as_bytes())
-                }
-            }
```

Comment:
We need to keep this.  It isn't a migration.  If the TransactionVersion is different, then the Proposals might not be compatible anymore and can't safely be used, so they need to be removed.